### PR TITLE
OHRM5X-1796: Create buzz - entities

### PR DESCRIPTION
**Pull request by RajithaKumara**

### CODE REVIEW

File: src/plugins/orangehrmBuzzPlugin/entity/BuzzComment.php [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+     *
+     * @ORM\ManyToOne(targetEntity="OrangeHRM\Entity\Employee")
+     * @ORM\JoinColumn(name="employee_number", referencedColumnName="emp_number")
+     */
+    private Employee $employee;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(name="number_of_likes", type="integer", nullable=true)
```
Comment (SandeepaOHRM) [inline src/plugins/orangehrmBuzzPlugin/entity/BuzzComment.php:59]: Just asking. If there is a default value for this field as 0, is there any point of making it nullable?

File: src/plugins/orangehrmBuzzPlugin/entity/BuzzComment.php [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+     * @var string
+     *
+     * @ORM\Column(name="comment_text",  type="string", nullable=true)
+     */
+    private string $text;
+
+    /**
+     * @var DateTime
+     *
+     * @ORM\Column(name="comment_time", type="datetime")
```
Comment (SandeepaOHRM) [inline src/plugins/orangehrmBuzzPlugin/entity/BuzzComment.php:73]: Seems text is required. If so, do we need to make this field nullable?

File: src/plugins/orangehrmBuzzPlugin/entity/BuzzComment.php (hunk 1 above)
Comment (RajithaKumara) [reply src/plugins/orangehrmBuzzPlugin/entity/BuzzComment.php:59]: nice catch. about to define it in next PR. for the moment thought to keep it

File: src/plugins/orangehrmBuzzPlugin/entity/BuzzComment.php (hunk 2 above)
Comment (RajithaKumara) [reply src/plugins/orangehrmBuzzPlugin/entity/BuzzComment.php:73]: no need to keep it nullable. I have revert it to make backward compatible with 4.x

File: src/plugins/orangehrmBuzzPlugin/test/Entity/BuzzPostTest.php [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+use OrangeHRM\Entity\BuzzPost;
+use OrangeHRM\Entity\Employee;
+use OrangeHRM\Tests\Util\EntityTestCase;
+use OrangeHRM\Tests\Util\TestDataService;
+
+/**
+ * @group Buzz
+ * @group Entity
+ */
+class BuzzPostTest extends EntityTestCase
```
Comment (ManulMax) [inline src/plugins/orangehrmBuzzPlugin/test/Entity/BuzzPostTest.php:33]: Will add test for other entities after merge

**Pull request merged**